src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discont36 = 20;
+  const discont7 = 50;
+  const rentPerDay = 40 * days;
```

Comment:
rentPerDay is 40. This value is the rent price for all days.
Create a variable for 40